ReduxTestTask/Classes/NewsList/View/NewsParentController.swift

```diff
@@ -0,0 +1,133 @@
+//
+//  NewsParentController.swift
+//  ReduxTestTask
+//
+//  Created by Polina Hill on 10/1/19.
+//  Copyright © 2019 Polina Hill. All rights reserved.
+//
+
+import UIKit
+import RxSwift
+import IGListKit
+
+final class NewsParentController: UIViewController {
+
+    struct Props {
+        let items: [NewsCell.Props]
+        let error: String?
+        let isLoading: Bool
+    }
+    
+    private lazy var contentView = NewsListView()
+    private var data = [ListDiffable]()
```

Comment:
This data should be in the view. 

Controller and view have different props. Like this:

Props {
  let viewProps: NewsListView.Props
  let error: String?
  let isLoading: Bool
}